lib/TextProcessing/HeadlineProvider.php

```diff
@@ -16,6 +16,7 @@ public function __construct(
 		private OpenAiAPIService $openAiAPIService,
 		private IConfig $config,
 		private IL10N $l10n,
+		private ?string $userId,
```

Comment:
Doesn't work yet, because text processing providers potentially run in a background job which doesn't have a session

Reply:
If the userId is null, the quota checking is omitted currently. The other option would be to disable scheduling all together for now until your core pr is merged -- if the admin wishes to enforce a quota with absolute certainty.

Reply:
> If the userId is null, the quota checking is omitted currently.

Cool! Just wanted to make sure you know that userId is most likely not set